v3/src/components/web-view/collaborator-utils.ts

```diff
@@ -0,0 +1,45 @@
+// Functions used by the Collaborative plugin
+
```

Comment:
I think this file deserves a comment, especially about the __editable__ attribute functionality.